### DISCUSSION TITLE
printer

### DISCUSSION THREAD

**Discussion by Blakechez** [maruohon/litematica#321] [Ideas]
make a printer even if its the most simple thing ever

**Comment by maruohon** [comment]
A printer will come as a separate extension mod at some point.

There are currently two 3rd party printer solutions, one as an extension mod, and one is fork of the mod.
* The extension mod is here: https://github.com/eatmyvenom/litematica-printer/releases
* and the fork is here: https://github.com/aleksilassila/litematica-printer/releases
